### PR TITLE
chore: using mget when get plugin for /apps

### PR DESCRIPTION
**Pull request by nquang29**
#1353

### CODE REVIEW

**Comment by nquang29** [comment]
Image 23-11-24 at 01 32

File: backend/database/redis_db.py [hunk 1]
```diff
@@ -137,6 +137,19 @@ def get_plugin_reviews(plugin_id: str) -> dict:
...
+    if not plugin_ids:
+        return {}
+
+    keys = [f'plugins:{plugin_id}:reviews' for plugin_id in plugin_ids]
+    reviews = r.mget(keys)
+    if reviews is None:
+        return {}
+    return {
+        plugin_id: eval(review) if review else {}
+        for plugin_id, review in zip(plugin_ids, reviews)
```
Comment (beastoin) [inline backend/database/redis_db.py:150]: what happen if a plugin does not have any review, does zip(x,y) still work as expected?

Comment (nquang29) [reply backend/database/redis_db.py:150]: i tested and it worked
keys: [plugin1, plugin2, plugin3]
plugin2 dont have review -> mget will return: [review1, None, review2]
-> this code above will return: {plugin1: review1, plugin2: {}, plugin3: review3}

**Comment by beastoin** [comment]
lgtm @nquang29 🥳

**Comment by beastoin** [comment]
can you find any other function with a loop to infras like this one?

**Comment by nquang29** [comment]
i will find and have a MR to fix these issues

**Pull request merged**